geode-core/src/distributedTest/java/org/apache/geode/management/QueryDataDUnitTest.java

```diff
@@ -313,7 +314,8 @@ public void testErrors() {
           distributedSystemMXBean.queryData(regionsNotFoundOnMembersQuery, member1.getId(), 2);
       assertThat(regionsNotFoundOnMembersResult, isJson(withJsonPath("$.message",
           equalTo(
-              String.format("Cannot find regions %s in specified members", "/" + regionName)))));
+              String.format("Cannot find regions %s in specified members",
+                  SEPARATOR + regionName)))));
```

Comment:
Nitpick: since we're using `String.format` here, wouldn't it be better (performance wise) to include and extra parameter within the `String` instead of the concatenation?.

Reply:
Scratch that, it's just a test...